cssdb.json

```
@@ -526,6 +526,15 @@
       }
     ]
   },
+  {
+    "id": "read-only-write",
+    "title": "`:read-only` and `:read-write` selectors",
+    "description": "Pseudo-classes to match elements which are considered user-alterable",
+    "specification": "https://drafts.csswg.org/selectors-4/#rw-pseudos",
```

This has advanced from csswg.org to w3.org.

```suggestion
    "specification": "https://www.w3.org/TR/selectors-4/#rw-pseudos",
```